### PR TITLE
Add FastAPI icon

### PR DESCRIPTION
**Pull request by uselessscat**
**Issue:** #3521
**Alexa rank:** 55,319

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
To make this icon I used the SVG available [in their repo](https://github.com/tiangolo/fastapi/blob/master/docs/en/docs/img/icon-white.svg).

I used the color from [this SVG](https://github.com/tiangolo/fastapi/blob/master/docs/en/docs/img/logo-teal-vector.svg)

### CODE REVIEW

**Comment by NovaGL** [comment]
Hi @uselessscat

Your icon failed due to it not being the right size or centered. This can be fixed by using a tool such as Inkscape to see if everthing is correct or increasing the precision when using SVGOMG.

**Comment by uselessscat** [comment]
@NovaGL Fixed

File: _data/simple-icons.json [hunk 1]
```diff
@@ -1980,6 +1980,11 @@
             "hex": "000000",
             "source": "https://www.farfetch.com/"
         },
+        {
+            "title": "FastAPI",
+            "hex": "009688",
+            "source": "https://github.com/tiangolo/fastapi/blob/6205935323ded4767438ee81623892621b353415/docs/en/docs/img/icon-white-bg.png"
```
Comment (runxel) [inline _data/simple-icons.json:1986]: Why do you link to a raster file?
You even linked to the correct svg files in your PR text... 🤔

**Review by runxel — CHANGES_REQUESTED** [review]
All the linked icons are a circle, your version is not. How so?

File: _data/simple-icons.json (hunk 1 above)
Comment (uselessscat) [reply _data/simple-icons.json]: Hi @runxel, I updated the source :)

**Comment by uselessscat** [comment]
@runxel The original icon looks like:

image

The icon I uploaded doesn't have the circle because of this preview:

simpleicons2

If i add the circle it looks like (The colors are inverse):

simpleicons

The previews was generated with https://petershaggynoble.github.io/MDI-Sandbox/simpleicons/preview/ (This link was in the default text when I opened the PR)

But i can rollback to https://github.com/simple-icons/simple-icons/blob/eddd3eb3cd61db6e4d14f442ad19b64b09368430/icons/fastapi.svg if you consider that it is correct.

**Comment by runxel** [comment]
> So I assumed that the surrounding circle was not important.

Well we cannot just change their logo because we think it should look like something else 😅

**Review by uselessscat — COMMENTED** [review]
I updated the icon to match the original svg version

**Comment by runxel** [comment]
Wonderful, thank you! 💯

**Review by runxel — APPROVED** [review]

**Pull request merged**